Samples/SimpleNet7FunctionWithCustomRuntime/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Information": [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
+    "To learn more about the Lambda commands with the .NET Core CLI execute the following command at the command line in the project root directory.",
+    "dotnet lambda help",
+    "All the command line options for the Lambda command can be specified in this file."
+  ],
+  "profile": "",
+  "region": "",
+  "configuration": "Release",
+  "function-runtime": "provided.al2",
+  "function-memory-size": 256,
+  "function-timeout": 30,
+  "function-handler": "bootstrap",
+  "msbuild-parameters": "--self-contained true"
```

Comment:
Are you sure that you need to publish Native AOT with self-contained parameter? As far as I know, in .NET 7, it will do nothing for Native AOT applications and is not required.

Reply:
Good point! I think it's still an error in .NET 6 if you don't have it. But for .NET 7 it isn't required anymore. I'll remove it from the .NET 7 examples.